go.mod

```diff
@@ -0,0 +1,38 @@
+module academy_bootcamp
```

Comment:
Your module should be named the same way you have it stored in a code repository (github, gitlab, etc.); you should follow the same path.

In this case, should be `github.com/FernandoGal25/academy-go-q42021`; this is done this way because other people might import your package, and avoid conflicts if another package is exactly named as you have it right now (not probable).